packages/sprotty/src/base/views/view.tsx

```diff
@@ -155,8 +156,20 @@ export class EmptyView implements IView {
  */
 @injectable()
 export class MissingView implements IView {
+    private static positionMap = new Map<string, Point>();
+
     render(model: Readonly<SModelElementImpl>, context: RenderingContext): VNode {
-        const position: Point = (model as any).position || Point.ORIGIN;
-        return <text class-sprotty-missing={true} x={position.x} y={position.y}>?{model.id}?</text>;
+        const position: Point = (model as any).position || this.getPostion(model.type);
+        return <text class-sprotty-missing={true} x={position.x} y={position.y}>missing "{model.type}" view</text>;
```

Comment:
Currently only a few examples include CSS for the `sprotty-missing` class. I think we should move this to the main sprotty.css file. I'd also include some text alignment property so the messages are aligned left instead of centered.